src/VirtoCommerce.ImageToolsModule.Data/Services/DefaultImageService.cs

```diff
@@ -31,17 +68,23 @@ public DefaultImageService(IBlobStorageProvider storageProvider, ILogger<Default
         /// <returns>Image object.</returns>
         public virtual async Task<Image<Rgba32>> LoadImageAsync(string imageUrl)
         {
-            _logger.LogInformation($"Loading image {imageUrl}");
+            _logger.LogInformation("Loading image {imageUrl}", imageUrl);
```

Comment:
Why did we leave the curled brackets here - {imageUrl}? The same question at line 85 below.